tests/__init__.py

```diff
@@ -0,0 +1 @@
+# This test suite requires the icommands packages. for the moment.
```

Comment:
how about...

"For now, this test suite requires the iCommands packages."

Reply:
done